### DISCUSSION TITLE
帮忙看看routes报错

### DISCUSSION THREAD

**Discussion by lvtu1124** [umijs/umi#8520] [Q&A]
![9_J~`Y_Q J693 P_XO`O(9V](https://user-images.githubusercontent.com/34616532/178753395-becabab6-f0f9-42f9-b2bb-47b387384da2.png)

![V8X(GIW5D@18PH5KRJ`_)KJ](https://user-images.githubusercontent.com/34616532/178753458-08b7ed28-5373-4ac4-87c9-827cc60618dc.png)

![O5LQM XPCU(~8EWC5RA JT](https://user-images.githubusercontent.com/34616532/178753592-9d1666ed-a895-4a5a-aa84-33cbf4c8bfaa.jpg)

![973E2~QKI`AN%ZF684DP2CF](https://user-images.githubusercontent.com/34616532/178753657-3dda1d7c-f867-49c5-a702-b5a5c0b7571d.png)

umi3还正常，umi4会报错

**Comment by fz6m** [comment]
配置文件不在应用范围里，alias `@` 用不了。
